tests/testdata/programs/dotnet/dashed_module/tf_files/mymod.tf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
related to my comment above, I think maybe these aren't necessary to check in?

Reply:
It helps eye-ball the generated TF files for specific operations when reviewing, especially when it is relevant to what the PR does. We can discuss it the next time around, I personally prefer having them but if others on the team feel otherwise, could only use it for testing.